streams/src/test/java/org/apache/kafka/streams/tests/StreamsBrokerDownResilienceTest.java

```diff
@@ -104,8 +107,9 @@ public void uncaughtException(final Thread t, final Throwable e) {
         Runtime.getRuntime().addShutdownHook(new Thread(new Runnable() {
             @Override
             public void run() {
-                System.out.println("Shutting down streams now");
-                streams.close(10, TimeUnit.SECONDS);
+                streams.close(30, TimeUnit.SECONDS);
+                System.out.println("Shutting down streams resilience test app now");
```

Comment:
nit: Shutting down -> Complete shutdown ..

Also in the verification we can change accordingly.

Reply:
ack